CHANGES.md

```diff
@@ -0,0 +1,15 @@
+Dec 2023 - PR #3
+Hardware
+- Parts
+	- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
+	- Added slip clutch and slip clutch pulley CAD 
+	- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
+- Drawings
+	- Added drawings of updated CAD
+
+Docs
+- Updated BOM:
+	- Updated CAD components
+	- Changed the material of the takeup_rest from aluminum to stainless steel
+	- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck
```

Comment:
```suggestion

- Updated CAD components
- Changed the material of the takeup_rest from aluminum to stainless steel
- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck

```
Would you expand `Updated CAD components` above?

Reply:
Yeah! 
```suggestion
- Changed the reel drive plate and slip clutch shaft CAD file reference to reel_drive_clamp_slipclutch and shaft_adatper_slipclutch
- Removed components for old slipclutch
- Added current slipclutch
- Changed the material of the takeup_rest from aluminum to stainless steel
- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck
```